apps/builder/app/builder/features/style-panel/shared/shadow-content.tsx

```diff
@@ -93,6 +94,29 @@ const boxShadowInsetValues = [
   { value: "inset", Icon: ShadowInsetIcon },
 ] as const;
 
+const shadowPropertySyntaxes = {
```

Comment:
it feels like this mapping doens't make the code above simpler, maybe I am wrong, but have you checked if using `propertySyntaxes` directly everywhere leads to better readability? ignore this message if this is still better

Reply:
I personally believe this is much easier to read. Or else, we need to go through if checks every time we use these. As we are managing three properties `box-shadow`, `drop-shadow` and `text-shadow` using the same component.